app/src/components/Blog.jsx

```diff
@@ -1,17 +1,36 @@
 import React, { Component } from 'react';
 import Footer from './Shared/Footer';
 import { Link } from 'react-router';
-import listposts from '../../styles/components/c-list-posts.scss';
+import listPosts from '../../styles/components/c-list-posts.scss';
 import CoverPrimary from './Shared/CoverPrimary';
 import PaginationBlog from './Blog/Pagination';
 import boxtriangle from '../../assets/icons/box_triangle.svg';
 
 class Blog extends Component {
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      totalPageCount: 0
+    };
+
+    this.boundOnPageChange = this.onPageChange.bind(this);
```

Comment:
But again, confusing them don't really matter since we only mean to use one. If you go with 2 names, I'd say pageChangeBound instead of boundOnPageChange.